_sass/minima/layout/_examples.scss

```
@@ -1,10 +1,10 @@
 .edu {
-    background: linear-gradient(90deg, #09222E 0.01%, #154858 23.97%, #1C6173 48.08%, #164959 75%, #0A232F 98.44%);
-    padding: 80px 0px 20px 0px;
+    background: url('/graalpy.github.io/resources/img/pattern-top.svg') 70% 0%/1750px no-repeat, linear-gradient(90deg, #09222E 0.01%, #154858 23.97%, #1C6173 48.08%, #164959 75%, #0A232F 98.44%);
```

This should not hardcode the base url. You should do this:

1. Add a YAML frontmatter to the scss file and forward the base url as a SASS variable
```
---
$baseurl: "{{ site.baseurl }}";
---
```
2. Then in this background spec, use 
```suggestion
    background: url('#{$baseurl}/resources/img/pattern-top.svg') 70% 0%/1750px no-repeat, linear-gradient(90deg, #09222E 0.01%, #154858 23.97%, #1C6173 48.08%, #164959 75%, #0A232F 98.44%);
```